sale_order_invoice_amount/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2021 ForgeFlow S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
```

Comment:
Tiny detail, IMO it is better to remove copyrights from init files, less places to have copy-paste errors :stuck_out_tongue_closed_eyes: